docker-compose.yml

```diff
@@ -1,14 +1,50 @@
-version: '3'
-
 services:
+  postgres_db:
+    image: postgres:16.4-bullseye
+    environment:
+      - POSTGRES_USER=${POSTGRES_USER}
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+      - POSTGRES_DB=${POSTGRES_DB}
+    volumes:
+      - postgres:/var/lib/postgresql/data
+    networks:
+      - internal_network
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready"]
```

Comment:
this will choose 'postgres' username by default, so better to get it from .env too?

`pg_isready -U ${POSTGRES_USER}`